Cargo.toml

```diff
@@ -23,6 +23,7 @@ exclude = ["tests", ".github"]
 serde = { version = "1.0", optional = true, default-features = false }
 core = { version = "1.0.0", optional = true, package = "rustc-std-workspace-core" }
 compiler_builtins = { version = "0.1.2", optional = true }
+zerocopy = { version = "0.6.1", optional = true, default-features = false }
```

Comment:
I think we should do something like:

```toml
zerocopy_0_6 = { version = "0.6.1", optional = true, default-features = false, package = "zerocopy" }
```

here and remove the explicit feature. Otherwise users can depend on `zerocopy` directly, which will become confusing when a new version is released.